migrateToAutomaticPackageRestore.ps1

```diff
@@ -7,6 +9,7 @@ $listOfBadStuff = @(
 	"\s*<Import Project=""\$\(SolutionDir\)\\\.nuget\\NuGet\.targets"".*?/>",
 	"\s*<Target Name=""EnsureNuGetPackageBuildImports"" BeforeTargets=""PrepareForBuild"">(.|\n)*?</Target>"
 	"\s*<RestorePackages>\w*</RestorePackages>"
+    "\s*<SolutionDir Condition=""\$\(SolutionDir\) == '' Or \$\(SolutionDir\) == '\*Undefined\*'"">\.\.\\</SolutionDir>"
```

Comment:
I can't accept this pull request because of this line. Some other tools might need this information to work! It also does not break automatic package restore.